scripts/publish.sh

```diff
@@ -10,13 +10,16 @@ if [ -z "$VERSION" ]; then
   exit 1
 fi
 
```

Comment:
I like to add `set -eu -o pipefail` to bash scripts to make them stop executing on any error